lib/rpc.ts

```diff
@@ -268,7 +272,7 @@ export class Rpc extends EventEmitter {
 
   private _makeCall(iface: string, meth: string, args: any[], resultChecker: tic.Checker,
                     fwdDest: string): Promise<any> {
-    return new Promise((resolve, reject) => {
+    return this._callWrapper(() => new Promise((resolve, reject) => {
```

Comment:
I prefer to avoid big anonymous functions: how about we rename the original `_makeCall` to `_makeCallRaw`, and add a new `_makeCall` like you have here, with `return this._callWrapper(() => this._makeCallRaw(iface, meth, args, resultChecker, fwdDest))`?